src/android/PerfChart.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { PerfherderGraphContainer } from '../utils/PerfherderGraphContainer';
+import { COMBOS, PLATFORMS } from './config';
+import { selectFrom } from '../vendor/vectors';
+import { TimeDomain } from '../vendor/jx/domains';
+
+const timeDomain = new TimeDomain({ past: '3month', interval: 'day' });
+
+const PerfChart = ({ title, suite }) => PLATFORMS.map(({ label, filter, id }) => (
+  <Grid item xs={6} key={id}>
+    <PerfherderGraphContainer
+      timeDomain={timeDomain}
+      title={`${title} ${label}`}
+      series={selectFrom(COMBOS)
+        .where({ suite: `${suite}` })
+        .map(({ browserLabel, filter: browserSuiteFilter }) => ({
+          label: browserLabel,
+          filter: {
+            and: [{ missing: 'test' }, filter, browserSuiteFilter],
```

Comment:
you can put the `{missing: "test"}` in the suite filters.  or you can sneak it in the `COMBO` construction.

```
const COMBOS = COMBO_TABLE.data.map(row => {missing:"test", ...Data.zip(COMBO_TABLE.header, row)});
```

Reply:
@klahnakoski Tried sneaking it into `COMBO` construction, did not work for me. Instead, I updated the `COMBO_TABLE`. Works fine. Thanks for the heads up.